NWNForBG/Setup-NWNForBG.tp2

```diff
@@ -6125,26 +6125,47 @@ REPLACE ~XXX104456~ @104456
 REPLACE ~XXX104457~ @104456
 
 // worldmap
-ACTION_IF FILE_EXISTS ~Worldmap/map_mods_areas.tbl~ THEN BEGIN
-  COPY ~Worldmap/map_mods_areas.tbl~  ~Worldmap~
-   APPEND_FILE ~NWNForBG/Worldmap/areas.tbl~
-
-  //preliminary step - making LANGUAGE temporary file until the new WeiDU
-  COPY ~NWNForBG/tra/autotra/%LANGUAGE%/worldmap.tra~  ~tmp_worldmap.tra~
-  COPY ~Worldmap/map_mods_trans.tra~  ~Worldmap~
-   APPEND_FILE ~tmp_worldmap.tra~
-
-  ACTION_IF FILE_EXISTS ~Worldmap/map_mods_links.tbl~ THEN BEGIN
-    COPY ~Worldmap/map_mods_links.tbl~  ~Worldmap~
-     APPEND_FILE ~NWNForBG/Worldmap/links.tbl~
-  END ELSE BEGIN
+//NEW: Force it on original installations, skip it for EE
+ACTION_IF GAME_IS ~bg2 tob bgt~ THEN BEGIN
+  ACTION_IF FILE_EXISTS ~Worldmap/map_mods_areas.tbl~ THEN BEGIN

```

Comment:
You could create function to wrap worldmap select and then invoke the function from NWNForBG/lib/worldmapselect.tph script to not duplicate the same code here and in tph file.

Reply:
Great job! Thanks!